tests/components/tedee/test_sensor.py

```diff
@@ -0,0 +1,38 @@
+"""Tests for the Tedee Sensors."""
+
+
+from unittest.mock import MagicMock
+
+import pytest
+from syrupy import SnapshotAssertion
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr, entity_registry as er
+
+pytestmark = pytest.mark.usefixtures("init_integration")
+
+
+@pytest.mark.parametrize(
+    "key",
+    [
+        "battery",
+        "pullspring_duration",
+    ],
+)
```

Comment:
I've learnt yesterday that parametrizing costs a bit more than just a normal for loop, so can we maybe just do this as a for loop in these tests?

Reply:
we can (and I just committed like that), the only thing to keep in mind is, that snapshots become harder to read as they have .1, .2 in their name and not the parameter